cfme/cloud/keypairs.py

```diff
@@ -0,0 +1,43 @@
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.common import SummaryMixin, Taggable
+from cfme.web_ui import Quadicon, flash
+from functools import partial
+from cfme.web_ui import toolbar as tb
+from utils.wait import wait_for
+
+
+cfg_btn = partial(tb.select, "Configuration")
+
+
+class KeyPair(Taggable, SummaryMixin):
+    """ docstring for KeyPairs """
+
+    def __init__(self, name=None):
+        self.name = name
+
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div[contains(., "{} (Summary)")]'.format(self.name))
```

Comment:
I would recommend using cfme.web_ui.summary_title and/or cfme.web_ui.breadcrumbs to detect this :)